tests/EdgeToEdge/Routes/ShowMembershipConfirmationRouteTest.php

```diff
@@ -101,9 +101,28 @@ public function testCallOnAnonymizedRecord_deniedPageIsShown(): void {
 				]
 			);
 
-			$this->assertContains( 'access_denied_membership_confirmation', $client->getResponse()->getContent() );
-			$this->assertSame( 403, $client->getResponse()->getStatusCode() );
+			$this->assertAccessIsDenied( $client );
 		} );
 	}
 
+	public function testCallOnUnknownApplicationId_deniedPageIsShown(): void {
+		$this->createEnvironment( [], function ( Client $client, FunFunFactory $factory ): void {
+			$client->request(
+				Request::METHOD_GET,
+				self::PATH,
+				[
+					'id' => self::UNKNOWN_APPLICATION_ID,
+					'accessToken' => self::WRONG_ACCESS_TOKEN
```

Comment:
Shouldn't this be `CORRECT_ACCESS_TOKEN` to avoid the impression of testing the wrong thing/two things at once? Not that it matters in the implementation because the key === id+access token ...